test/unit/models/embassy_test.rb

```diff
@@ -1,49 +1,12 @@
 require "test_helper"
 
 class EmbassyTest < ActiveSupport::TestCase
-  setup do
```

Comment:
Minor: Can you explain *why* you've moved the `setup` into the relevant tests?

Reply:
I've now added additional explanation in the commit message and force-pushed.